apps/scripts/Bubble.c

```diff
@@ -0,0 +1,43 @@
+void swap(int *xp, int *yp)
+{
+    int temp = *xp;
+    *xp = *yp;
+    *yp = temp;
+}
+ 
+// A function to implement bubble sort
+void bubbleSort(int arr[], int n)
+{
+    int i, j;
+    for (i = 0; i < n-1; i++)    
+        for (j = 0; j < n-i-1; j++)
+            if (arr[j] > arr[j+1])
+                swap(&arr[j], &arr[j+1]);
+}
+ 
+#define MMIO_GENERAL  (*(volatile int (*)[64])(0x00400F00))//
```

Comment:
1. The value is different from the one in AliveTest.c. Is that intentional?
2. Syntax should be simplified (see above)
3. Perhaps this macro should be moved into a common header for all source files

Reply:
1.typo in Alive Test
2.done
3.maybe later